dbe/domain-eng.yml

```diff
@@ -133,10 +133,10 @@ responses:
       - text: |
           Reply with one of the words in BOLD to continue
 
-          *ACCEPT* ✅ - _to accept the terms and conditions_
-          *MORE* 📄 - _to read the terms and conditions_
+          ACCEPT ✅ - _to accept the terms and conditions_
+          MORE 📄 - _to read the terms and conditions_
 
-          (or Reply *MENU* 📌 _to return to the main menu_)
+          (or Reply MENU 📌 _to return to the main menu_)"
```

Comment:
The double quote at the end looks out of place.

Reply:
Yes thanks. When I copy from the spreadsheet to my editor it adds `"`, I missed this one.